vars/kola.groovy

```diff
@@ -157,7 +165,7 @@ def call(params = [:]) {
             def id = marker == "" ? "kola-upgrade" : "kola-upgrade-${marker}"
             ids += id
             try {
-                runKola(id, 'run-upgrade',  "--upgrades")
+                runKola(id, 'run-upgrade',  "--upgrades ${skipKolaTagsArgs}")
```

Comment:
I think let's keep out upgrade tests from this for now. It currently doesn't run for RHCOS and those tests should instead be converted into regular kola tests (see also discussions in https://github.com/coreos/coreos-assembler/issues/3714).

Reply:
Will do, I wasnt sure if this would be required for the upgrade tests. I'll remove it.